Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -2400,7 +2400,8 @@ void WebAutomationSession::takeScreenshot(const Inspector::Protocol::Automation:
 #if PLATFORM(GTK) || PLATFORM(COCOA)
     Function<void(WebPageProxy&, std::optional<WebCore::IntRect>&&, Ref<TakeScreenshotCallback>&&)> takeViewSnapsot = [](WebPageProxy& page, std::optional<WebCore::IntRect>&& rect, Ref<TakeScreenshotCallback>&& callback) {
         page.callAfterNextPresentationUpdate([page = Ref { page }, rect = WTFMove(rect), callback = WTFMove(callback)] () mutable {
-            auto snapshot = page->takeViewSnapshot(WTFMove(rect));
+            const bool alwaysUseNonHardwareCapturePath = true;
```

Comment:
We would normally make `takeViewSnapshot` take an enum class parameter with `::Yes`/`::No`, rather than introduce a `const bool` here.